x/sequencers/types/permission.go

```diff
@@ -24,19 +24,15 @@ func (p *PermissionList) Validate() error {
 	if len(p.Permissions) == 0 {
 		return errors.New("permissions field cannot be empty")
 	}
```

Comment:
would still suggest removing this
empty permissions is fine

Reply:
@danwt My own opinion, if the permission is empty, we should not set it to kv store. I also made a check in RevokePermission to delete the empty permission address.